### ISSUE TITLE
set messagerate -1 doens't exist

### ISSUE THREAD

**Issue by khancyr** [ArduPilot/ardupilot_wiki#4137]
see here : https://github.com/ArduPilot/ardupilot_wiki/blob/master/dev/source/docs/mavlink-requesting-data.rst

**Comment by khancyr** [comment]
no that is just not the right command on mavproxy to  disable the streamrate  handling :
```
STABILIZE> set messagerate -1
STABILIZE> Unknown setting 'messagerate'
```
fixed in https://github.com/ArduPilot/ardupilot_wiki/pull/4142

**Comment by Hwurzburg** [comment]
sorry I was the one confused about which -1 you were talking about...merged #4142
